packages/buttons/src/elements/Anchor.tsx

```diff
@@ -10,22 +10,15 @@ import PropTypes from 'prop-types';
 import { StyledAnchor, StyledExternalIcon } from '../styled';
 
 export interface IAnchorProps extends AnchorHTMLAttributes<HTMLAnchorElement> {
-  /** Apply danger styling */
+  /** Applies danger styling */
   isDanger?: boolean;
   /**
-   * Used when the anchor navigates to an external resource. Applies `target="_blank"`
-   * along with `rel="noopener noreferrer"` to ensure safe
-   * [cross-origin destination links](https://web.dev/external-anchors-use-rel-noopener/).
-   **/
+   * Applies anchor navigation to an external resource. When enabled, target="_blank" and
+   * rel="noopener noreferrer" attributes are added to ensure safe [cross-origin destination links](https://web.dev/external-anchors-use-rel-noopener/).
```

Comment:
[nit] I really want these to establish a line break precedent. Please add a newline after the word "safe". Keep run-on lines (past 80-100 column mark) shorter, where possible.

Reply:
Fixed in latest push.